sonarqube/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN cp -a /opt/sonarqube/data /opt/sonarqube/data-init && \
 	cp -a /opt/sonarqube/extensions /opt/sonarqube/extensions-init && \
 	chown root:root /opt/sonarqube && chmod -R gu+rwX /opt/sonarqube
 ADD plugins.sh /opt/sonarqube/bin/plugins.sh
+ADD https://github.com/rht-labs/sonar-auth-openshift/releases/latest/download/sonar-auth-openshift-plugin.jar /opt/sonarqube/extensions-init/plugins/
```

Comment:
Do we want to point to latest? or a pinned version? typically, i'd suggest a pinned version

Reply:
I kinda like keeping the latest at the moment while the plugin is maturing. I'm open to adding a pinned version if necessary. Also committed to not introducing breaking changes with v1.x of the plugin.